src/main/java/org/tasks/reservation/CoworkingSpaceBooking.java

```diff
@@ -0,0 +1,20 @@
+package org.tasks.reservation;
+
+public class CoworkingSpaceBooking {
+    private final CoworkingSpace coworkingSpace;
```

Comment:
why its final?)

Reply:
I used final, because this field doesn't have any setters and it can be only set via constructor.